lib/hybridDB.ts

```
@@ -296,6 +309,52 @@ export class HybridDB {
     return LocalDB.getThreads();
   }
 
+  // Load priority threads (pinned and project threads) from remote
+  static async loadPriorityThreadsFromRemote(): Promise<Thread[]> {
+    if (this.isGuestMode) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)